watergun_assassin/authentication/tests.py

```diff
@@ -0,0 +1,54 @@
+from django.test import TestCase
+from django.urls import reverse
+from datetime import datetime
+
+
+class UserCreationTest(TestCase):
+    def test_valid_payload(self):
+        url = reverse('create_user')
+        username = 'admin'
+        password = 'kjlkjlkjlk123'
```

Comment:
Can we use something like `fakePassword` for the value here, rather than `kjlk...`. The random string is a bit distracting when looking through the code.